skaffold.yaml

```diff
@@ -923,14 +923,14 @@ metadata:
 build:
   tagPolicy:
     customTemplate:
-      template: "{{.version}}-{{.sha}}"
+      template: "{{.version}}-{{.timestamp}}"
       components:
         - name: version
           envTemplate:
             template: "{{.GARDENER_VERSION}}"
-        - name: sha
-          gitCommit:
-            variant: AbbrevCommitSha
+        - name: timestamp
+          dateTime:
+            format: "20060102150405"
```

Comment:
I'm unsure/unhappy about this change as it deviates from all other Skaffold tag policies.
Skaffold would produce the same build tag in the development loop as both the Gardener version and commit hash stay the same. However, the gardenlet deployment should restart when a new image has been built and pushed by the dev-loop. As Skaffold is not taking care of the deployment in the operator setup the timestamp causes the deployment to restart.

Reply:
This change would affect all the gardenlet tags even when you run `make gardener-up`. That looks weird.
Couldn't you introduce a now `operator-dev` profile for `gardenlet` which changes the tag for the dev loop purpose only?